applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -572,6 +575,8 @@ static void audio_datapath_i2s_blk_complete(uint32_t frame_start_ts, uint32_t *r
 
 	/********** I2S TX **********/
 	static uint8_t *tx_buf;
+#if (CONFIG_STREAM_BIDIRECTIONAL || (CONFIG_AUDIO_DEV == HEADSET))
```

Comment:
In general it is correct to include this #if here, and we might want to have it. The problem is that we get somewhat of a domino effect of #if throughout the audio_datapath.c file.
It can be argued that this #if is not that important, since the gateway usually has more power available (compared to e.g. earbuds).